lib/api/website.js

```diff
@@ -53,7 +53,7 @@ function _errorActions(err, errorDocument, routingRules,
                 // eslint-disable-next-line no-param-reassign
                 request.objectKey = errorDocument;
                 if (!isObjAuthorized(bucket, errObjMD, request.apiMethods || 'objectGet',
-                    constants.publicId, null, log, request, request.actionImplicitDenies)) {
+                    constants.publicId, null, log, request, request.actionImplicitDenies, true)) {
```

Comment:
I forgot this one in my last PR https://github.com/scality/cloudserver/pull/5517/files#diff-bf6d6accaa9226cb0ba471ae47c24a517c0927609a306ea8c63caee9b4610db8